wordpress-mobile/WordPress-iOS
# Prevent FAB to be obscured by notices

Fixes #14442

To test:

* Enter the "Customize Your Site" screen from your new site
* Tap "Set your site title" or "Upload a site icon"
* Verify that the FAB is visible and not obscured by the notice
* Additionally, tap on any option in "Customize Your Site" and verify that the FAB behaves the same way

##### Note
There are currently different behaviors of the notice and spotlight when tapping on different cells of the tableView (for example, tapping on "Blog Posts" and going back, spotlight and notice are gone, tapping on "Site Pages" the notice stays on screen and going back will still show the spotlight).
These will be fixed with issue #14441

PR submission checklist:
